### PR TITLE
Issue 1084 - Projects permissions

### LINKED ISSUE #1084: User Management: Projects permissions
### Description
"Assign Permissions" view for projects should be redesigned to can easily set permissions for specific or multiple users.
https://sketch.cloud/s/mrZ88/all/3d-repo-monday/projects_permissions

### Goals
- [x] Admins can change project, whose permissions should be modified
- [x] Admins can set permissions for single user
- [x] Admins can set permissions for multiple users
- [x] Admins can filter user permissions list by name

### Tasks
- [x] User permissions list component
- [x] Button linked to "Model and federations permissions" view
- [x] Footer with description

### PR DESCRIPTION
**Pull request by simplypixi**
This fixes #1084 

I had to create this branch from #1082 branch. #1082 PR is not merged yet, but I need these changes to continue my work.

#### Description
- Added project's permissions list component
- Added button, which redirects to "Model and federations permissions" view
- Added footer with proper description
- Added `services` directory for shared services

### CODE REVIEW

**Comment by simplypixi** [comment]
> Project admins cannot access the project tab because the teamspace is not listed. to reproduce you will need a 2nd login (feel free to reset password for anyone on that database):

How can I reset password? ;)

**Comment by simplypixi** [comment]
> footer is massive for some screens:
@carmenfan Are you sure that the footer is so big? Footer doesn't have a border, so maybe it depends on scroll position. I think that border of last visible row is hidden because of `overflow: hidden`. Could you re-test this bug? ;)

**Comment by carmenfan** [comment]
@simplypixi on admin database, do this https://docs.mongodb.com/manual/reference/method/db.changeUserPassword/

**Comment by simplypixi** [comment]
@carmenfan I fixed the reported bugs ;)
@Charence Is my code ok?

File: frontend/components/permissions-list/pug/permissions-list.pug [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				layout="column"
+				layout-align="center center"
+				flex="50"
+				ng-repeat="permission in vm.PROJECT_PERMISSIONS")
+				div(layout="row" layout-align="center center")
+					list-sort-button(
+						ng-disabled="!vm.processedPermissions.length"
+						ng-class="{'active': vm.currentSort.type === vm.SORT_TYPES.FIELD && vm.currentSort.config.key === permission.key}"
+						on-change="vm.setSortType(vm.SORT_TYPES.FIELD, order, {key: permission.key})")
+					.usersListCellLabel {{::permission.label}}
```
[the remark this answers is not in the document]
Comment (simplypixi) [reply frontend/components/permissions-list/pug/permissions-list.pug]: Yes ;)

**Comment by simplypixi** [comment]
@Charence I fixed the bugs

**Pull request merged**